resources/windows/wix/project.wxs.erb

```diff
@@ -55,16 +55,22 @@
     <!-- INSTALLDIR -->
     <CustomAction Id="SaveCmdLineInstallDir" Property="CMDLINE_INSTALLDIR" Value="[INSTALLDIR]" Execute="firstSequence" />
     <CustomAction Id="SetFromCmdLineInstallDir" Property="INSTALLDIR" Value="[CMDLINE_INSTALLDIR]" Execute="firstSequence" />
+    <%= File.read("resources/windows/CustomActions") if File.exist?("resources/windows/CustomActions") -%>
+    <%- if @platform.architecture == "x86" -%>
+      <%= File.read("resources/windows/x86-CustomActions") if File.exist?("resources/windows/x86-CustomActions") -%>
+    <%- end -%>
 
     <InstallUISequence>
```

Comment:
We should take this out of the project file altogether and put it in the project specific (e.g. puppet-agent) files.